### PR TITLE
gh-118671: Updated dead ActiveState links

### LINKED ISSUE #118671: Dead link for `linspace` recipe
# Documentation

https://github.com/python/cpython/blob/00d913c6718aa365027c6dcf850e8f40731e54fc/Doc/library/stdtypes.rst?plain=1#L1497-L1501

The "See also" section for `range` seems to have a dead link to the linspace recipe: https://code.activestate.com/recipes/579000/

### Linked PRs
* gh-118730
* gh-118754

### PR DESCRIPTION
**Pull request by trag1c**
Closes #118671

* Issue: gh-118671

----
📚 Documentation preview 📚: https://cpython-previews--118730.org.readthedocs.build/

### CODE REVIEW

**Comment by Eclips4** [comment]
I think this kind of change does not need a "NEWS" entry. Can you remove it? (I will add the "skip news" label to satisfy CI.)

File: Doc/howto/urllib2.rst [hunk 1]
```diff
@@ -594,5 +594,5 @@ This document was reviewed and revised by John Lee.
        scripts with a localhost server, I have to prevent urllib from using
        the proxy.
 .. [#] urllib opener for SSL proxy (CONNECT method): `ASPN Cookbook Recipe
-       <https://code.activestate.com/recipes/456195/>`_.
+       <https://code.activestate.com/recipes/456195-urrlib2-opener-for-ssl-proxy-connect-method/>`_.
```
Comment (Eclips4) [inline Doc/howto/urllib2.rst:597]: We should keep the diff as minimal as possible. Therefore, there is no need for a new line. You can keep it on one line.
No worries about the length of the line, our doc has a lines that are more longer than this :)
(This applies to the rest of the PR)

**Comment by Eclips4** [comment]
TSAN failure is unrelated - see #118729 for more details

File: Doc/howto/urllib2.rst (hunk 1 above)
Comment (trag1c) [reply Doc/howto/urllib2.rst:597]: Oh, good to know :smile:

**Review by Eclips4 — APPROVED** [review]
All of the new URLs are working. Thank you 👍

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**